src/Everscale/WorkchainType.h

```diff
@@ -6,11 +6,8 @@
 
 #pragma once
 
-namespace TW::Everscale {
-
-enum WorkchainType {
-    Masterchain = -1,
-    Basechain = 0,
-};
+#include "CommonTON/WorkchainType.h"
 
-} // namespace TW::Everscale
+namespace TW::Everscale {
+    using WorkchainType = CommonTON::WorkchainType;
+} // namespace TW::Everscale
```

Comment:
new line break

Reply:
> new line break

There is a line break, but GitHub doesn't show it (as well as alert about "no newlines" though)